frontend/taipy-gui/src/utils/formatConversion.ts

```diff
@@ -96,24 +90,14 @@ export const sprintfToD3Converter = (format: string) => {
     });
 }
 
-/**
```

Comment:
I don't know that I would have removed the comment alltogether.
It could be useful in the future...
Why not simply replate /** by /*?

Reply:
resolved